dejagnu/baseboards/arc-sim-nsimdrv.exp

```diff
@@ -30,9 +30,16 @@ load_generic_config "sim"
 load_base_board_description "basic-sim"
 
 # Any multilib options are set in an environment variable.
-set multilib_opts "$env(ARC_MULTILIB_OPTIONS)"
+if { [ info exists env(ARC_MULTILIB_OPTIONS) ] } {
+    set multilib_opts "$env(ARC_MULTILIB_OPTIONS)"
+} else {
+    set multilib_opts ""
```

Comment:
`multilib_opts` is not needed anymore with other changes of this patch, so `process_multilib_options` can ne directly invoked from inside the if-else. I also wonder if it makes sense to invoke `process_multilib_options ""`? If it is not needed than a simple "if" can be enough:
```
if { [ info exists env(ARC_MULTILIB_OPTIONS) ] } {
    process_multilib_options "$env(ARC_MULTILIB_OPTIONS)"
}
```